frontend/src/queries/nodes/InsightViz/InsightViz.scss

```diff
@@ -0,0 +1,109 @@
+.InsightViz {
```

Comment:
Replaces the `Insight.scss`. Much more specific styling with `InsightContainer` renamed to `InsightVizDisplay` which makes the heirarchy clearer